cmake/projects/ceres-solver/hunter.cmake

```diff
@@ -0,0 +1,29 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+# List of versions here...
+hunter_add_version(
+    PACKAGE_NAME
+    ceres-solver
+    VERSION
+    "1.12.0"
+    URL
+    "https://github.com/hunter-packages/ceres-solver/archive/hunter.zip"
```

Comment:
This archive is not "stable", after you push updates to branch `hunter` it will change. Use `tar.gz` archive from releases: https://github.com/hunter-packages/ceres-solver/releases/tag/1.12.0rc1